### DISCUSSION TITLE
Undo/Redo for SuperTextField

### DISCUSSION THREAD

**Discussion by matthew-carroll** [Flutter-Bounty-Hunters/super_editor#515] [Milestones]
Add support for undo/redo to SuperTextField.

### FUNDING
**Looking for funding**

**Comment by AQ4Life1** [comment]
Has there been any progress on this feature?
Thank you!

**Reply by matthew-carroll** [reply]
Turtle shifted their focus, so they won't be funding this work. Therefore, we're looking for any other funders that might want or need undo/redo support for `SuperTextField`.

We will be delivering undo/redo support for `SuperEditor` with funding from a few companies.

**Reply by manoletre** [reply]
Is there any progress on the funding? / on this feature?

I could be potentially interested in funding this, but would like some additional information on costs / how the funding structure works. Is there any guide on that anywhere?

**Reply by matthew-carroll** [reply]
Hi @manoletre you can see our standard marketing material at www.flutterbountyhunters.com

If you're still interested after reading that, I can work with you directly for the next steps.

There may be anywhere from 1-3 other companies that would join you in funding undo/redo for SuperTextField.

Do you also need undo/redo for SuperEditor?
